template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Program.Designer.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Company.WebApplication1
+{
+    public class DesignTimeServiceProviderFactory : IDesignTimeServiceProviderFactory
```

Comment:
Hmm... It would require `Program` to not be `static` and always have a default constructor (or one with `args`)